h2o-core/src/main/java/hex/ModelMojoWriter.java

```diff
@@ -241,4 +244,14 @@ private void writeDomains() throws IOException {
       finishWritingTextFile();
     }
   }
+
+  /** Create file that contains model details in JSON format.
+   * This information is pulled from the models schema.
+   */
+  private void writeModelDetails() throws IOException{
+    ModelSchemaV3 modelSchema = (ModelSchemaV3) SchemaServer.schema(3, model).fillFromImpl(model);
```

Comment:
Still thinking about this line - it brings a connection to REST layer. It is not bad, since we would like to have uniform output, on other hand, it is error prone and depends on registration of REST API layer. See PR-800.